src/server/api/controllers/users.controller.js

```diff
@@ -19,6 +19,23 @@ const getRole = async (req) => {
   }
 };
 
+// get user's name for a specific uid
+const getName = async (req) => {
```

Comment:
How is this going o be used? By admins?

Reply:
@dpfernandes 👆

Reply:
We need it for logout popup. 
<img width="238" alt="Screenshot 2020-04-11 at 18 51 02" src="https://user-images.githubusercontent.com/10253713/79049703-76622880-7c25-11ea-99c3-0d2c4dc8584d.png">

Reply:
And the name is not extractable on the client side?

Reply:
We'll be getting the name of the currently logged-in user to display on some button in the profile :)

Reply:
Ok. 
But maybe it would make more sense to only have the id in the mysql table and then the rest of the information like name and email in the firebase table...

Reply:
Name is only in DB, in Firebase is email

Reply:
Yes and is there a specific reason for having division?

Reply:
Yes, the decision was to not use any of Firebase DB, only MYSQL.. therefore we can't add user's name to Firebase.

Reply:
The Firebase authentication db is already in use. Otherwise we can't retrieve tokens from it. The username can be extracted on the client side through the property user.displayName. 
As of now I can't see why the email should be kept in mysql and the name in the firebase db.

Reply:
or the other way around :)

Reply:
Confusion has a tendency to build up if left unhandled.

Reply:
If the name is stored in mysql when the user signs up, then we will have to make a post request and a mysql request to add the name in addition to this one to get the name. Those can both be avoided if the name is added on the client side on sign up

Reply:
Agree Niels, but where should we add name to Firebase? This is the adding user screen: 
<img width="929" alt="Screenshot 2020-04-11 at 21 38 05" src="https://user-images.githubusercontent.com/10253713/79053365-15dee580-7c3d-11ea-8fb6-7c92ee765b85.png">
And this is user.displayName
<img width="364" alt="Screenshot 2020-04-11 at 21 39 46" src="https://user-images.githubusercontent.com/10253713/79053375-30b15a00-7c3d-11ea-9f47-be0b87548b62.png">
Thanks!

Reply:
Im thinking with an addition input field for the name

Reply:
I found this https://stackoverflow.com/questions/43509021/how-to-add-username-with-email-and-password-in-firebase

Reply:
I think you should simply go to the documention. It is perfectly readable

Reply:
Sure. We don't have signup form, I am really not sure how to add user's name to Firebase. I found in the documentation how to update profile, and change displayName... but I still have to have name for it.

Reply:
Not sure i follow... You shouldn't have to have a name to add a name. That would be impossible. As far as I can see it is a matter of invoking the updateProfile method

Reply:
Niels, we don't have signup form. Which name should be written as displayName in the updateProfile method? 
user.updateProfile({
  displayName: "User's name",
})
We added users manually to Firebase, so that we can use Firebase auth. There are only name and password fields, as in the screenshot I sent you.

Reply:
Well if we don't want a signup page, we would just have to have an input field where the user can set the name. You could for instance set the email as the displayName by default and then allow the user to change it later

Reply:
Fine with me, if you think that's better. But we should communicate with @dpfernandes about it.

Reply:
I'm curious, how would you have done it otherwise? Added the names manually to the mysql database?

Reply:
Also, how are the users of the website going to signup, without a signup page?

Reply:
Users table is already there, with id, uid, name, email etc

Reply:
"We created a table and therefore have to use it" 
Thats not a valid argument

Reply:
For now we will use the hardcoded **User** values but in the future we should look into adding all the **User data on Firebase** and by doing this we will look into the **Create a new user/Signup flow.**